docs/DeveloperGuide.md

```diff
@@ -363,6 +385,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   * Steps 1a1-1a2 are repeated until the employee ID entered is valid.
   * Use case resumes from step 2.
 
+**Use case: UC4 - Finding an employee**
+
+**MSS**
+
+1. User requests to find an employee with name and surname
+2. EmployeeManager finds all employees whose names contain either the full name or surname keyword 
+3. EmployeeManager updates the displayed list with the found employees.
+3. Use case ends.
+
+**Extensions**
+
+* 1a. The user did not use a valid predefined prefix to search for employee by name
+    * 1a1. EmployeeManager informs user that the input is invalid.
+    * 1a2. User enters new input and new prefix.
+    * Steps 1a1-1a2 are repeated until the increment amount entered is valid.
```

Comment:
Need to check with this extension again, it seems like it's for increment feature not find feature.

Reply:
ok! Thanks!